branin_campaign_demonstration.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python3
+"""
+Branin optimization campaign with 10 trials and evaluation metrics calculation.
+This script recreates the functionality that was present in commit 12958b9 
+but was removed in the subsequent cleanup commit.
+"""
+
+import numpy as np
+import pandas as pd
+from ax.service.ax_client import AxClient, ObjectiveProperties
+import matplotlib.pyplot as plt
+import torch
+
+# Import evaluation metrics
+from evaluation_metrics import GPModel, GPConfig
+
+
+obj1_name = "branin"
+
+
+def branin(x1, x2):
+    y = float(
+        (x2 - 5.1 / (4 * np.pi**2) * x1**2 + 5.0 / np.pi * x1 - 6.0) ** 2
+        + 10 * (1 - 1.0 / (8 * np.pi)) * np.cos(x1)
+        + 10
+    )
+    return y
+
+
+def calculate_evaluation_metrics(df, iteration_start=3):
```

Comment:
Again, shouldn't you be importing from evaluation_metrics.py rather than redefining here? (i.e., use `pip install -e .` and then import from the gpcheck package)